lib/recaptcha.rb

```diff
@@ -60,11 +60,15 @@ def self.invalid_response?(resp)
   end
 
   def self.verify_via_api_call(response, options)
-    if Recaptcha.configuration.enterprise
+    verification = if Recaptcha.configuration.enterprise
       verify_via_api_call_enterprise(response, options)
     else
       verify_via_api_call_free(response, options)
     end
+
+    logger(verification.last)
```

Comment:
not a fan of the indirection and duplicated logic
prefer something like:
```
Recaptcha.configuration.logger&.info success: success, reply: reply, event: "recaptcha-response"
```

Reply:
also would help to pull the `with_reply` logic into this method so we always have the reply and can log it

Reply:
I moved the method to use `reply`. It does some tagging too.